manifest-base.json

```
@@ -10,24 +10,24 @@
   "title": "A32NX",
   "dependencies": [
     {
-      "package_version": "0.1.58",
+      "package_version": "0.1.62",
       "name": "asobo-aircraft-a320-neo"
     },
     {
-      "package_version": "0.1.52",
+      "package_version": "0.1.58",
       "name": "asobo-vcockpits-instruments-a320-neo"
     },
     {
-      "package_version": "0.1.53",
+      "package_version": "0.1.58",
       "name": "asobo-vcockpits-instruments-airliners"
     },
     {
-      "package_version": "0.1.66",
+      "package_version": "0.1.69",
       "name": "fs-base-aircraft-common"
     }
   ],
   "content_type": "AIRCRAFT",
-  "total_package_size": "00000000000221782906",
-  "minimum_game_version": "1.9.5",
+  "total_package_size": "00000000000325682532",
+  "minimum_game_version": "1.10.5",
```

```suggestion
  "minimum_game_version": "1.10.8",
```
Current game version is 1.10.8